invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -563,6 +588,22 @@ def get_rights(self, obj):
                 entry["rightsUri"] = link
             serialized_rights.append(entry)
 
+        # Adding access_status information
```

Comment:
This can be removed, since it was addressed in https://github.com/inveniosoftware/invenio-rdm-records/pull/2047